common/djangoapps/student/models.py

```diff
@@ -753,6 +753,15 @@ def user_post_save_callback(sender, **kwargs):
             ceas = CourseEnrollmentAllowed.for_user(user).filter(auto_enroll=True)
 
             for cea in ceas:
+                # skip enrolling already enrolled users
+                if CourseEnrollment.is_enrolled(user, cea.course_id):
+                    # link the CEA to the user if the user didn't exist at the time the email address
```

Comment:
i think people may not know what a cea is, so maybe we can explain what this linkage is doing in this comment

Reply:
resolved in video conversation